code/style.css

```diff
@@ -1,7 +1,318 @@
+  margin: auto;
+  display: block;
+  padding: 50px;
+  opacity: 0.7;
+}
+/* Big quote styling */
+.quote {
+  grid-column-start: 1;
+  grid-column-end: 3;
+  background-color: #889189ad;
+  display: block;
+  align-content: center;
+  padding: 10vh;
+}
+.quote h3 {
+  font-size: 40px;
+  color: #f1ecec;
+  font-style: italic;
+  text-align: center;
+  font-family: "Abril Fatface";
+  font-weight: 100;
+}
+.quote h2 {
+  text-align: center;
+  color: rgb(216, 206, 233);
+  text-transform: uppercase;
+  font-size: 16px;
+}
+/* Newsletter signup section styling */
+.newsletter {
+  margin: auto;
+  padding: 20px;
+  background-color: white;
+  width: 60%;
+  max-height: 80%;
+  overflow: scroll;
+  box-shadow: 2px 2px 4px #525f4d;
+}
+.newsletter-image img {
+  /* Here I flipped the image horizontally to fit the design better */
+  -webkit-transform: scaleX(-1);
+  transform: scaleX(-1);
+  width: 80%;
+  height: auto;
+  margin: auto;
+  display: block;
+  padding: 50px;
+  opacity: 0.7;
+}
+.signup-form {
+  display: grid;
+  justify-content: center;
+  padding: 2vw;
+}
+.signup-form h3 {
+  font-family: "Merriweather", serif;
+  font-size: 30px;
+  font-style: italic;
+  color: #3e3f4b;
+  padding-bottom: 10px;
+  margin-bottom: 5px;
+}
+.signup-form > * {
+  font-size: 14px;
+}
+form input {
+  margin-bottom: 10px;
+  border: double 4px #b6a9c2;
+  border-radius: 10px;
+  padding: 5px 0;
+  background-color: rgb(243, 234, 253);
+}
+.radio {
+  margin-bottom: 10px;
+}
+.consent a {
+  text-decoration: none;
+  color: #7d6692;
+}
+.consent a:hover {
+  color: #dc8aec;
+}
+.submit {
+  width: 200px;
+  justify-self: center;
+  padding: 0.5em 3em;
+  border: 0.16em solid #c0d6c5;
+  margin: 10px 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #5f7565;
+  text-align: center;
+  transition: all 0.15s;
+  background-color: #aadfb693;
+  border-radius: 15px;
+}
+.submit:hover {
+  color: #dc8aec;
+  border-color: #e8c1f0;
+}
+.submit:active {
+  color: #bbbbbb;
+  border-color: #bbbbbb;
+}
+/* Footer styling */
+footer {
+  display: flex;
+  justify-content: space-between;
+  height: 250px;
+  background-color: #3e3f4b;
+}
+.footer-left {
+  font-size: 14px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px 0 10px 20px;
+  color: #9f94a8;
+}
+.logo-2 {
+  color: rgba(255, 255, 255, 0.226);
+  font-size: 120px;
+  font-family: "Abril Fatface";
+  padding-left: 30px;
+}
+.footer-links {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  border-left: 2px solid white;
+  padding-right: 80px;
+  padding-left: 80px;
+}
+.footer-links a {
+  text-decoration: none;
+  color: #dedcdc;
+}
+.footer-links a:first-of-type {
+  color: rgb(224, 252, 230);
+}
+.footer-links a:hover {
+  text-decoration: none;
+  color: #e8c1f0;
+}
+/* Tablet */
+@media (max-width: 1023px) and (min-width: 768px) {
+  .newsletter {
+    width: 80%;
+  }
+}
+/* Mobile */
+@media (max-width: 766px) {
```

Comment:
Your page is well responsive to the changes in mobile view and it's a clean, nice look where you only view the hero image, faq accordion, quote, newsletter form and footer!